scripts/publish.sh

```diff
@@ -14,10 +14,10 @@ newVersion=$1
 echo -e $BLUE"Publishing a new" $newVersion "version"$NC
 
 git checkout main && \
-  npm version $newVersion && \
-  npm run build:lib && \
-  cd ./lib && \
-  npm publish && \
-  cd ../ &&\
-  git push --follow-tags &&\
-  npm run deploy:docs
+yarn version --minor/major/patch && \
```

Comment:
I'm not great with shell scripts so this syntax might be wrong, but I think we need to do something like `yarn version --$newVersion` so that it interpolates the version passed in 🤔  The script that worked for me when I was working on deploying my changes was `yarn version --minor`

Reply:
aaaah thanks! I was suuuuper wondering about that, I'll fix 😁